File: OpenMP/hello1millsimd.c
#include <stdio.h> 
#include "omp.h"
int main(void) 
{ 
   char greetings[] = "Hello world!"; 
   int a;
   #pragma omp simd
   for ( a = 0; a < 1000000; a = a + 1 )
   	{
	printf("%s\n", greetings); 
   	}
return 0;
}

File: OpenMP/hello1millsimd.f90
program hello
	implicit none 
        include "omp_lib.h"
	character(len=16) :: greetings 
	integer :: a
	greetings = "Hello World!" 

	! Hello World, one million times
	!$omp simd
	do a = 1, 1000000
	print *, greetings   
	end do

end program hello

